root/scripts/talker/mechanic.txt

```diff
@@ -4922,6 +4922,19 @@ Rule C4M4EllisStoryMechanic
 	Response C4M4EllisStoryMechanic
 }
 
+Response C4M5EllisStoryMechanic
+{
```

Comment:
Will worry about this when Kerry can implement an engine criteria for chapters. Please remain patient.